config.json

```diff
@@ -1004,6 +1004,11 @@
       "slug": "booleans",
       "name": "Booleans"
     },
+    {
+      "uuid": "1721552b-9c55-432d-a776-05e618c25bb4",
+      "slug": "calling-functions-through-function-pointers",
+      "name": "Calling Functions Through Function Pointers"
```

Comment:
How about just "Function pointers" ?

Reply:
Well, I took it directly from your document. 😄 I like that it's specific, but I don't feel strongly about it.

Reply:
😄 

Both approaches have fairly clear meaning I think. The main advantage of "Function pointers" is that it is shorter.

This helps a couple of things:

1. Rendering on the site in different forms, e.g. in lists, tables, etc (especially on mobile)
2. Maintenance of the repository (less typing for maintainers!)

Reply:
Good points. Okay, it's all renamed.

Reply:
What's left to do: another review or just squash and merge?